Kickstarter-iOS/Locales/Base.lproj/Localizable.strings

```diff
@@ -429,8 +441,10 @@
 "Tap_the_heart_on_each_project_page_to_save_it" = "Tap the heart on each project page to save it. We’ll remind you 48 hours before the project ends.";
 "Tap_the_star_on_a_project_to_get_notified" = "Tap the star icon on a project to get notified 48 hours before it ends.";
 "Tap_the_star_on_each_project_page_to_save_it" = "Tap the star on each project page to save it. We’ll remind you 48 hours before the project ends.";
+"Ten_is_just_the_beginning" = "Ten is just the beginning.";
```

Comment:
I didn't merge the KSR10 stuff to master so that I didn't have to remove it again 😄  I'm working on improving our strings script so that we only generate strings for they keys that we want to.